### DISCUSSION TITLE
rate changes

### DISCUSSION THREAD

**Discussion by exylophone** [KadeArchive/Kade-Engine#2041] [Q&A]
im probably being dumb but how do you change the rate of normal week songs in base ke 1.7? do you have to edit files or is there a keybind for it?
![image](https://user-images.githubusercontent.com/87776554/131234862-970448e1-19b2-402a-8d93-d6abb0c4bbab.png)

**Comment by Spel0** [comment]
Ctrl + left/right arrow
